client/src/app/ui/modules/file-list/components/file-list/file-list.component.html

```diff
@@ -105,16 +105,20 @@
     </div>
 
     <!-- Indicator column -->
-    <div *osScrollingTableCell="'indicator'; row as mediafile; config: { width: 40 }" class="fill">
-        <div
-            [matMenuTriggerFor]="singleMediafileMenu"
-            [matMenuTriggerData]="{ mediafile: mediafile }"
-            [matTooltip]="tooltipFn(mediafile)"
-        >
-            <mat-icon class="foreground-icon" *ngIf="mediafile.isFont() && isUsedAsFontFn(mediafile)">
+    <div *osScrollingTableCell="'indicator'; row as mediafile; config: { width: 40 }" class="fill clickable">
+        <div>
```

Comment:
The additional wrapping div could be removed.

Reply:
Done.